src/api/operations/client.queries.graphql

```diff
@@ -74,6 +74,31 @@ query GetClientEnrollments(
   }
 }
 
+query GetClientEnrollmentWithHousehold(
+  $id: ID!
+  $limit: Int = 1
+  $offset: Int = 0
+  $filters: EnrollmentsForClientFilterOptions
```

Comment:
(If we end up keeping this approach) I'd suggest making `openOnDate` and `projectId` required variables, since that's the only intended use-case for this query. This resolves so much, it could get really expensive if it was resolved on a larger set of enrollments. I think we are _assuming_ that it shouldn't return a ton of enrollments, because a client shouldn't have multiple open enrollments at a given project on a given date-- but thats only true if those args are passed.  (Even though I see this defaults to `limit: 1`, it could still be used in other ways)